DevelopmentTools/Rakefile

```diff
@@ -99,8 +99,8 @@ end
 
 desc 'Update the changelog, no commit made'
 task :changelog do
-  puts "Updating changelog #{ENV['UPLIFT_GITHUB_TOKEN']}"
-  sh 'github_changelog_generator' if ENV['UPLIFT_GITHUB_TOKEN']
+  puts 'Updating changelog'
+  sh "github_changelog_generator -t #{ENV['UPLIFT_GITHUB_TOKEN']}" if ENV['UPLIFT_GITHUB_TOKEN']
```

Comment:
Does this need to be shown fully?

Reply:
This is actually to make up for the implementation. github_changelog_generator requires the CHANGELOG_GITHUB_TOKEN by default, so we need to input our own to be trusted